pkg/transform/oauth/github.go

```diff
@@ -59,6 +59,9 @@ func buildGitHubIP(serializer *json.Serializer, p IdentityProvider, config *conf
 	secretName := p.Name + "-secret"
 	idP.GitHub.ClientSecret.Name = secretName
 	secretContent, err := fetchStringSource(github.ClientSecret, config)
+	if err != nil {
+		return nil, nil, nil, err
```

Comment:
If you're returning 3 nils, I have to suggest wrapping up all the values in a type that captures the return value of a function. This isn't a blocking issue for this PR, but we should consider what this fn is actually returning and maybe abstract it.

Reply:
Feels like buildGitHubIP should return the thing it built or an error.